package.json

```
@@ -3,6 +3,10 @@
   "version": "1.1.4",
   "description": "Really Fast Deep Clone",
   "main": "index.js",
+  "exports": {
+    ".": "./index.js",
+    "./default": "./default/index.js"
```

I personally would just call the file `default.js`:

```suggestion
    "./default": "./default.js"
```